internal/utils/aggregator.go

```diff
@@ -99,6 +100,10 @@ func filter_valid_csv_records(main_df dataframe.DataFrame) (dataframe.DataFrame,
 		dataframe.F{Colname: "cpu_usage_container_max", Comparator: series.GreaterEq, Comparando: 0},
 		dataframe.F{Colname: "cpu_usage_container_min", Comparator: series.GreaterEq, Comparando: 0},
 		dataframe.F{Colname: "cpu_usage_container_avg", Comparator: series.GreaterEq, Comparando: 0},
+		dataframe.F{Colname: "owner_kind", Comparator: series.Neq, Comparando: ""},
+		dataframe.F{Colname: "owner_name", Comparator: series.Neq, Comparando: ""},
```

Comment:
@patilsuraj767  Do we need to check for both of these as well? Could you share some sample values?

Reply:
Yes, you can check row 104 in our sample CSV - https://github.com/RedHatInsights/ros-ocp-backend/blob/main/scripts/samples/ros-ocp-usage.csv#L104

Reply:
BTW I thought on this again and I realised that we need `owner_kind` and `owner_name` both but we do not need `workload` and `workload_type` so removing it.

Reply:
Meant to ask if these values are missing as well along with workload and workload_type, as per the cost card

Reply:
Meant to ask if these values are missing as well along with workload and workload_type, as per the cost card